report.php

```diff
@@ -91,6 +91,8 @@
             $defaults[$element] = $formelements->$element;
         }
     }
+
+    $default['resend'] = XAPI_REPORT_RESEND_FALSE;
```

Comment:
Can you remove this line so I can merge the PR?

I tested it locally and I'm still getting the same behaviour

Reply:
I have removed this.